libs/wingc/src/type_check.rs

```diff
@@ -341,6 +341,17 @@ impl Subtype for Type {
 				// TODO: Hack to make anything's compatible with all other types, specifically useful for handling core.Inflight handlers
 				true
 			}
+			(Self::Function(_), Self::Interface(r0)) => {
+				// TODO: Hack to make functions compatible with interfaces
+				// Remove this after https://github.com/winglang/wing/issues/1448
+				// Compare the function to a method on the interface named "handle" if it exists
+				if let Some(method) = r0.get_env().try_lookup("handle", None) {
+					let method = method.as_variable().unwrap();
+					self.is_subtype_of(&*method.type_)
```

Comment:
Since this is probably a temp hack, I'd do something along the lines of this to hack the `this` first arg and not change the function comparison below.
```suggestion
					let temp_fn = Self::Function(FunctionSignature { parameters: (vec![other]).concat(l0.parameters.clone()), ..l0.clone()});
					temp_fn.is_subtype_of(&*method.type_)
```

Reply:
Done